webapp/app/plugins/annotations/main.html

```diff
@@ -1,7 +1,7 @@
-<div ng-controller="AnnotationsPluginCtrl" class="row annotations-panel">
```

Comment:
This `row` class was just applying a -ve margin of 15px, so I removed it. I don't think there are any ill effects.